english/lightsaber-falls/lightsaber-falls.md

```diff
@@ -0,0 +1,9 @@
+You slice through the marshmallow man with your lightsaber. 
+Luckily for you, the heat from it cooks the marshmallows to perfection.
+You now have a tasty snack and a way out of the house. 
+
+But as you hurry to smoosh through the slowly closing hole in the wall, the lightsaber clatters to the floor.
+
+[You leap out nimbly and gaze back at the weapon](https://github.com/LaGracia/create-your-own-adventure/blob/lightsaber-falls/english/lightsaber-falls/wampa.md).
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.